tests/src/serverTestMultiMappedPV.cpp

```diff
@@ -0,0 +1,31 @@
+#define BOOST_TEST_MODULE serverTestMultiMappedPV
+#include <boost/test/included/unit_test.hpp>
+
+#include "DoocsAdapter.h"
+#include "serverBasedTestTools.h"
+#include <ChimeraTK/ControlSystemAdapter/Testing/ReferenceTestApplication.h>
+#include <doocs-server-test-helper/doocsServerTestHelper.h>
+
+extern const char* object_name;
+#include <doocs-server-test-helper/ThreadedDoocsServer.h>
+
+using namespace boost::unit_test_framework;
+using namespace boost::unit_test;
+using namespace ChimeraTK;
+
+DOOCS_ADAPTER_DEFAULT_FIXTURE_STATIC_APPLICATION
+
+/// Check that all expected variables are there.
+BOOST_AUTO_TEST_CASE(testMultiMapped, *boost::unit_test::tolerance(0.0001)) {
+  GlobalFixture::referenceTestApplication.runMainLoopOnce();
```

Comment:
Why do you run the main loop at all? This is only the "to app" direction, and you don't even check that it arrives there.